packages/typespec-client-generator-core/test/types/built-in-types.test.ts

```diff
@@ -3,17 +3,29 @@ import { AzureCoreTestLibrary } from "@azure-tools/typespec-azure-core/testing";
 import { expectDiagnostics } from "@typespec/compiler/testing";
 import { ok, strictEqual } from "assert";
 import { beforeEach, describe, it } from "vitest";
+import { leftOver } from "../../src/interfaces.js";
```

Comment:
is there a way that we could verify this without exporting it?
I exported this type only for test, and the type is a never, no one is ever going to use it and it is never useful to anyone